app-modules/interaction/src/Filament/Concerns/HasManyMorphedInteractionsTrait.php

```diff
@@ -100,17 +101,28 @@ public function table(Table $table): Table
             ->defaultSort('end_datetime', 'desc')
             ->columns([
                 TextColumn::make('subject')
+                    ->description(fn ($record) => $record->initiative->name . ' (' . $record->driver->name . ')')
                     ->icon(fn ($record) => $record->is_confidential ? 'heroicon-m-lock-closed' : null)
                     ->tooltip(fn ($record) => $record->is_confidential ? 'Confidential' : null),
+                TextColumn::make('type.name')
+                    ->label('Type')
+                    ->toggleable()
+                    ->sortable(),
+                TextColumn::make('status.name')
+                    ->label('Status')
+                    ->toggleable()
+                    ->sortable(),
                 TextColumn::make('start_datetime')
                     ->label('Start Time')
-                    ->dateTime(),
-                TextColumn::make('end_datetime')
-                    ->label('End Time')
-                    ->dateTime(),
+                    ->dateTime()
+                    ->sortable(),
                 TextColumn::make('created_at')
                     ->state(fn ($record) => $record->end_datetime ? $record->end_datetime->diffForHumans($record->start_datetime, CarbonInterface::DIFF_ABSOLUTE, true, 6) : '-')
                     ->label('Duration'),
+                TextColumn::make('user.name')
+                    ->label('Created By')
+                    ->description(fn ($record) => $record->user->job_title)
+                    ->sortable(),
                 TextColumn::make('initiative.name')
                     ->toggleable(isToggledHiddenByDefault: true),
                 TextColumn::make('driver.name')
```

Comment:
Because Initivative and Driver are now underneath the Subject I think the intention is that their columns be removed. You may want to check with Product though.